src/test/java/stepDefinition/MyAccountStepDef.java

```diff
@@ -18,7 +20,7 @@
     public void userChangesField(String elementName, String extension, Map<String, String> data) {
         waitForPageLoaded();
         beforeEdit = getElementByName(elementName.concat(extension)).getAttribute("value");
-        data.forEach((key, value) -> getElementByName(key).clear());
+        data.forEach((key, value) -> clearTextField(getElementByName(key)));
         data.forEach((key, value) -> getElementByName(key).sendKeys(value));
```

Comment:
as you made wrappers for the others, why would you let **sendKeys()** like this?

Reply:
done, thank you!